storage/remote/client.go

```diff
@@ -66,10 +77,10 @@ var (
 			Namespace: namespace,
 			Subsystem: subsystem,
 			Name:      "read_request_duration_seconds",
-			Help:      "Histogram of the latency for remote read requests.",
+			Help:      "Histogram of the latency for remote read requests. Note that for streamed responses this is only the duration of the initial call and does not include the processing of the stream.",
```

Comment:
I wished we could actually move this to be full latency including receiving all bytes etc, but good enough for now I guess.